src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/ReferenceTokenCompositeSearchParam.sql

```diff
@@ -9,6 +9,9 @@
     ReferenceResourceVersion1 int NULL,
     SystemId2 int NULL,
     Code2 varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL,
+    CONSTRAINT PK_ReferenceTokenCompositeSearchParam PRIMARY KEY NONCLUSTERED (ResourceTypeId, SearchParamId, ReferenceResourceId1, Code2, ResourceSurrogateId)
```

Comment:
This overlaps with IX_ReferenceTokenCompositeSearchParam_ReferenceResourceId1_Code2.  Should the non-clustered index be unique too?